flutter-idea/src/io/flutter/survey/FlutterSurveyService.java

```diff
@@ -20,12 +20,11 @@
 public class FlutterSurveyService {
   private static final String FLUTTER_LAST_SURVEY_CONTENT_CHECK_KEY = "FLUTTER_LAST_SURVEY_CONTENT_CHECK_KEY";
   private static final long CHECK_INTERVAL_IN_MS = TimeUnit.HOURS.toMillis(40);
-  private static final String CONTENT_URL = "https://flutter.dev/f/flutter-survey-metadata.json";
+  private static final String CONTENT_URL = "https://docs.flutter.dev/f/dart-devtools-survey-metadata.json";
```

Comment:
The IDEs should still be using flutter-survey-metadata.json, so you'll just need to add the `docs.` here

Reply:
Thanks, updated.